iron-dropdown.html

```diff
@@ -145,6 +145,10 @@
             type: Object
           },
 
+          focusTarget: {
```

Comment:
Also, I'm not super sure what it's being used for. I would pass a focus target? Is it a refid?